pkg/api/http.go

```diff
@@ -21,6 +23,7 @@ import (
 const (
 	URLParamTraceID = "traceID"
 	// search
+	urlParamQuery       = "q"
```

Comment:
I noticed Loki and Prometheus both use `query` instead. I think we opted for `q` since this is least likely to conflict with tag lookups using the old top-level query parameters.

Loki's API: https://grafana.com/docs/loki/latest/api/#query-loki
Prometheus' API: https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries

Reply:
I'm good on `q`